testfuzz/src/main/java/org/hyperledger/besu/testfuzz/EofContainerSubCommand.java

```diff
@@ -196,55 +216,54 @@ private void extractFile(final File f, final File initialCorpus) {
   public void fuzz(final byte[] bytes) {
     Bytes eofUnderTest = Bytes.wrap(bytes);
     String eofUnderTestHexString = eofUnderTest.toHexString();
-    Code code = evm.getCodeUncached(eofUnderTest);
-    Map<String, String> results = new LinkedHashMap<>();
-    boolean mismatch = false;
-    for (var client : externalClients) {
-      String value = client.differentialFuzz(eofUnderTestHexString);
-      results.put(client.getName(), value);
-      if (value == null || value.startsWith("fail: ")) {
-        mismatch = true; // if an external client fails, always report it as an error
-      }
-    }
-    boolean besuValid = false;
-    String besuReason;
-    if (!code.isValid()) {
-      besuReason = ((CodeInvalid) code).getInvalidReason();
-    } else if (code.getEofVersion() != 1) {
-      EOFLayout layout = EOFLayout.parseEOF(eofUnderTest);
-      if (layout.isValid()) {
-        besuReason = "Besu Parsing Error";
-        parentCommand.out.println(layout.version());
-        parentCommand.out.println(layout.invalidReason());
-        parentCommand.out.println(code.getEofVersion());
-        parentCommand.out.println(code.getClass().getName());
-        System.exit(1);
-        mismatch = true;
-      } else {
-        besuReason = layout.invalidReason();
-      }
-    } else if (EOFContainerMode.INITCODE.equals(
-        ((CodeV1) code).getEofLayout().containerMode().get())) {
-      besuReason = "Code is initcode, not runtime";
-    } else {
-      besuReason = "OK";
-      besuValid = true;
-    }
-    for (var entry : results.entrySet()) {
-      mismatch =
-          mismatch
-              || besuValid != entry.getValue().toUpperCase(Locale.getDefault()).startsWith("OK");
-    }
-    if (mismatch) {
-      parentCommand.out.println("besu: " + besuReason);
-      for (var entry : results.entrySet()) {
+
+    AtomicBoolean passHappened = new AtomicBoolean(false);
+    AtomicBoolean failHappened = new AtomicBoolean(false);
+
+    Map<String, String> resultMap =
+        fuzzingClients.stream()
+            .parallel()
+            .map(
+                client -> {
+                  Stopwatch stopwatch = Stopwatch.createStarted();
+                  String value = client.differentialFuzz(eofUnderTestHexString);
+                  stopwatch.stop();
+                  long elapsedMicros = stopwatch.elapsed(TimeUnit.MICROSECONDS);
+                  if (elapsedMicros > timeThresholdNs
```

Comment:
are we comparing microseconds with nanoseconds?

Reply:
Micros.  Only variable name was off.